icu4c/source/samples/datecal/cal.cpp

```diff
@@ -17,6 +17,8 @@
 #include "unicode/gregocal.h"
 #include <stdio.h>
 
+using namespace icu;
```

Comment:
Interesting that I didn't see any warnings when building the samples with MSVC...
regardless, thanks for finding and fixing this.

Reply:
I've been fixing other compile errors and warnings, too. Different compilers!